gps2val/gpsd_feeder.py

```diff
@@ -20,115 +20,276 @@
-    manual_config.add_argument("--insecure", help="For KUKSA.val server specify False, for KUKSA.val databroker there is currently no security so specify True; default: False", nargs='?' , default="False")
-    manual_config.add_argument("--certificate", help="Specify the path to your Client.pem file; default: Client.pem", nargs='?' , default="Client.pem")
-    manual_config.add_argument("--cacertificate", help="Specify the path to your CA.pem; default: CA.pem", nargs='?' , default="CA.pem")
-    manual_config.add_argument("--token", help="Specify the path to your JWT token; default: all-read-write.json", nargs='?' , default="all-read-write.json")
-    manual_config.add_argument("--file", help="Specify the path to your config file; default: config/gpsd_feeder.ini", nargs='?' , default="config/gpsd_feeder.ini")
-    manual_config.add_argument("--gpsd_host", help="Specify the host for gpsd to start on; default: 127.0.0.1", nargs='?' , default="127.0.0.1")
-    manual_config.add_argument("--gpsd_port", help="Specify the port for gpsd to start on; default: 2948", nargs='?' , default="2948")
-    manual_config.add_argument("--interval", help="Specify the interval time for feeding gps data; default: 1", nargs='?' , default="1")
+    manual_config.add_argument("--host",
+                               help="Specify the host where too look for KUKSA.val server/databroker; "
+                                    "default: 127.0.0.1",
+                               nargs='?', default="127.0.0.1")
+    manual_config.add_argument("--port",
+                               help="Specify the port where too look for KUKSA.val server/databroker; default: 8090",
+                               nargs='?', default="8090")
+    manual_config.add_argument("--protocol",
+                               help="If you want to connect to KUKSA.val server specify ws. "
+                                    "If you want to connect to KUKSA.val databroker specify grpc; default: ws",
+                                    nargs='?', default="ws")
+    manual_config.add_argument("--insecure",
+                               help="For KUKSA.val server specify False, "
+                                    "for KUKSA.val databroker there is currently no security so specify True; "
+                                    "default: False",
+                               nargs='?', default="False")
+    manual_config.add_argument("--certificate",
+                               help="Specify the path to your Client.pem file; default: Client.pem",
+                               nargs='?', default="Client.pem")
+    manual_config.add_argument("--cacertificate",
+                               help="Specify the path to your CA.pem; default: CA.pem",
+                               nargs='?', default="CA.pem")
+    manual_config.add_argument("--token",
```

Comment:
maybe we should change this default.

Reply:
We could change it to "" here, make sense as other defaults reflect databroker setup